kythe/cxx/indexer/cxx/KytheGraphObserver.cc

```diff
@@ -1257,21 +1257,21 @@ void KytheGraphObserver::recordDeprecated(const NodeId& NodeId,
 }
 
 void KytheGraphObserver::recordDiagnostic(const Range& Range,
-                                          std::string_view Signature,
-                                          std::string_view Message) {
+                                          const llvm::StringRef& Signature,
+                                          const llvm::StringRef& Message) {
   proto::VName anchor_vname = VNameFromRange(Range);
 
   proto::VName dn_vname;
-  dn_vname.set_signature(
-      std::string(absl::StrCat(anchor_vname.signature(), "-", Signature)));
+  dn_vname.set_signature(std::string(
```

Comment:
`absl::StrCat` returns `std::string`; the explicit conversion here is unnecessary

Reply:
Done.